web/components/Canvas/MainMenu/MainMenu.module.scss

```diff
@@ -1,28 +1,27 @@
 .MainMenu {
   display: flex;
   justify-content: center;
-  align-items: center;
-  height: 100%;
+  margin-top: 50%;

```

Comment:
prefer align items vs margin

Reply:
I completely get where you're coming from with structuring elements. Please correct me if I'm wrong, but in this case using `margin-top` and `align-items` will be accomplishing different things. `align-items: center` will center the contents of the card, vs `margin-top` will change spacing of the card against the canvas. We could use `align-items: center` in the canvas, but that's not what we are going to want when we switch to the Board view. Maybe we hard code a pixel margin against the canvas? Reading through [this](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items), I'm not seeing a way align-items would solve what we are trying to do here. Thoughts?